src/main/java/io/excaliburfrc/robot/subsystems/Drive.java

```diff
@@ -19,15 +43,112 @@
   private final CANSparkMax rightFollower =
       new CANSparkMax(DrivetrainConstants.RIGHT_FOLLOWER_ID, MotorType.kBrushless);
 
+  private final RelativeEncoder rightEncoder = rightLeader.getEncoder();
+  private final RelativeEncoder leftEncoder = leftLeader.getEncoder();
+
   private final DifferentialDrive drive = new DifferentialDrive(leftLeader, rightLeader);
 
+  private final DifferentialDriveKinematics driveKinematics =
+      new DifferentialDriveKinematics(DrivetrainConstants.TRACKWIDTH_METERS);
+
+  private final DifferentialDriveVoltageConstraint autoVoltageConstraint =
+      new DifferentialDriveVoltageConstraint(
+          new SimpleMotorFeedforward(kS, kV, kA), driveKinematics, 10);
```

Comment:
why are you constructing *another* ff object instead of passing in the existing one? not that this object should be here anyway